enhancements/machine-config/mco-cgroupsv2-support.md

```diff
@@ -0,0 +1,123 @@
+---
+title: CGroupsv2 Machine Config Pool Enablement
+authors:
+  - "@rphillips"
+reviewers:
+  - "@mrunalp"
+approvers:
+  - "@yuqi-zhang"
+  - "@mrunalp"
+creation-date: 2021-10-19
+last-updated: 2021-10-19
+status: implementable
+---
+
+# CgroupsV2 Machine Config Pool Enablement
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+cgroupsv2 enablement in Kubernetes has progressed to the point that it is
+usable. The underlying runtime (cri-o) and supporting subsystems are now ready
+for customers to begin their own testing with it. Not all workloads will be
+compatible with cgroupsv2, so it will *not* be enabled by default within
+Openshift at this time.
+
+## Motivation
+
+Migrating to Cgroupsv2 will bring in many new features and fixes not found in
+Cgroupsv1. Cgroupsv1 is considered 'legacy' and migrating to Cgroupsv2 is
+considered necessary since RHEL ships with Cgroupsv2 on by default. (Openshift
+4.x currently disables Cgroupsv2 in favor of Cgroupsv1).
+
+Some features of Cgroupsv2 include:
+
+* IO enhancments
+* User based OOM killer
+* Cgroup aware OOM killer
+
+[Kubernetes On Cgroupsv2 - Video](https://www.youtube.com/watch?v=u8h0e84HxcE)
+
+### Goals
+
+- [ ] Enable Cgroupsv2 on a Machine Config Pool
```

Comment:
This is only for new nodes (mentioned below) if so can we call that out a bit clearer here?